lib/terrafying/components/service.rb

```diff
@@ -70,10 +71,15 @@ def create_in(vpc, name, options = {})
           options = options.merge_with_arrays_merged(fluentd_conf)
         end
 
-        unless options.key? :user_data
+        if isignition == true
+         unless options.key? :user_data
           options[:user_data] = Ignition.generate(options)
+         end
+        else
+          unless options.key? :user_data && isignition == false
+            options[:user_data] = Cloudconfig.generate(options)
+          end
```

Comment:
Thoughts on changing the logic to be a bit more readable and compact?
Like;
```
unless options.key? :user_data
  if isignition == true
    options[:user_data] = Ignition.generate(options)
  else if isignition == false
    options[:user_data] = Cloudconfig.generate(options)
  end
end
```

Reply:
also can we change it to just `ignition` not `isignition`. `isignition` is hard to read